evals/tasks/write.py

```
@@ -0,0 +1,648 @@
+                project_id=project_id,
+                cycle_id=cur_id,
+                params=WorkItemQueryParams(per_page=100),
+            )
+            names = {(i.name or "").strip() for i in (on13.results or [])}
+            missing = [t for t in unfinished if t not in names]
+            if missing:
+                ok = False
+                notes.append(f"unfinished not on Sprint 13: {missing}")
+            else:
+                notes.append(f"{len(unfinished)} unfinished on Sprint 13")
+        except Exception as exc:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)